docs/apis/api_1.adoc

```diff
@@ -0,0 +1,390 @@
+= [big blue]*Alcor Controller APIs*
```

Comment:
Why do we need 3 API documents? What are the difference of doc 1, 2 and 3?

Reply:
Hello Liguang,

That little nicer needs css file, otherwise document is looking a little bit too simple.

Reply:
Sure thanks. Let us plan to remove it in the "ultimate" automated version.